### ISSUE TITLE
allow multiple splits does not work with test duplication

### ISSUE THREAD

**Issue by callumparr** [nanoporetech/duplex-tools#13]
I am trying to see if I can use this tool to split read concatamers that are generated from proximity ligation assay.

Before I went in too deep altering the underlying adapter sequences I wanted to see if it works well using the current adapters.

I took the fastq in the test/data and duplicated the sequence.

But running 

```
duplex_tools split_on_adapter fastq_in split_at_adapter_test/test_data2 Native --print_alignment --allow_multiple_splits --debug_output
```
the fastq sequence is as follows:

```
@200bases-tailhead-200bases
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
**CTGTCTCTTATACACATCTCCTGCAGG
AGATGTGTATAAGAGACAG**
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
**CTGTCTCTTATACACATCTCCTGCAGG
AGATGTGTATAAGAGACAG**
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
TGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGTGTCGTGATCATGCTAGTGTAGCTGATGTGCG
+
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
IIIIIIIIIIIIIIIIIIIIIIIIIII
IIIIIIIIIIIIIIIIIII
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
IIIIIIIIIIIIIIIIIIIIIIIIIII
IIIIIIIIIIIIIIIIIII
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
```

the output splits at the first tail-head adapter but skips over the second tail-head within the fastq.

**Comment by onordesjo** [comment]
Hi! Thanks for reporting, I'll take a look at it and see what may be going on.

**Comment by onordesjo** [comment]
No worries at all, possibly good to get this documented in case people stumble upon (hopefully really rare) issues with internal adapters very early/late in the reads! Just cleaning some things up, there should be two new params in the new version (0.2.10), trim_start, and trim_end. You would hopefully only need to change trim_end from default of 200 to something like 20.

**Comment by callumparr** [comment]
Just got round to testing this, despite reducing trim to 10 on start and end I can still only get one split on the test fastq included. 

```
duplex_tools split_on_adapter duplex-tools/tests/data/fastq_200-th-200-th-200 split_on_adapter_test/test_example/test_data12 Native --print_alignment --debug --allow_multiple_splits --trim_end 10 --trim_start 10
[14:21:08 - SplitOnAdapters] Duplex tools version: 0.2.10
0it [00:00, ?it/s]200bases-tailhead-200bases editdistance-5
GCAA-T-ACGNNNNNNNNNNNACTTCGTTCAGTTACGTATTGCT
|.||-|-|.-||||||||||||||||||||||||||||||||||
GTAACTGAA-CGAAGTAATGTACTTCGTTCAGTTACGTATTGCT
[14:21:08 - SplitOnAdapters] Split 1 reads
Kept 0 reads
[14:21:08 - SplitOnAdapters] Wrote a total of 2 reads
```

**Comment by onordesjo** [comment]
There, should be fixed now with v0.2.11:
https://github.com/nanoporetech/duplex-tools/releases/tag/v0.2.11

**Comment by callumparr** [comment]
Sorry for taking so long to get round to reply. This now performs as expected.
